lib/newrelic.js

```diff
@@ -67,36 +67,34 @@ class NewRelic {
      * @param {object} defaultMetrics default metrics to include in every New Relic event
      */
     constructor(options, defaultMetrics={}) {
-        const OPENWHISK_NEWRELIC_DISABLE_METRICS = process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS || false;
 
         this.defaultMetrics = Object.assign(Metrics.openwhisk(), defaultMetrics);
-
         this.canSendMetrics = false;
 
-        if (OPENWHISK_NEWRELIC_DISABLE_METRICS !== true) {
+        if (process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS === `true`) {
```

Comment:
`if (process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS)` should be enough, then. I don't think we really care about its contained. If it's not set we'll go into the else condition anyway.